src/Uno.UI/UI/Xaml/Internal/DependencyPropertyHelper.cs

```diff
@@ -0,0 +1,216 @@
+{
+	/// <summary>
+	/// Get the default value of a <see cref="DependencyProperty"/> for a given type.
+	/// </summary>
+	/// <remarks>
+	/// This is the value defined in the metadata of the property, which _may_ be overriden
+	/// using the internal DependencyProperty.OverrideMetadata() method on a per-type basis
+	/// -- that's why the type parameter is required.
+	///
+	/// A classic example of that is the Button.VerticalAlignmentProperty which is overridden
+	/// to be VerticalAlignment.Center, while the default value is VerticalAlignment.Stretch
+	/// from the base class.
+	/// </remarks>
+	public static object? GetDefaultValue(DependencyProperty dependencyProperty, Type forType)
+		=> dependencyProperty.GetMetadata(forType)?.DefaultValue;
+
+	/// <summary>
+	/// Get the default value of a <see cref="DependencyProperty"/> for a given type.
+	/// </summary>
+	/// <remarks>
+	/// This is the value defined in the metadata of the property, which _may_ be overriden
+	/// using the internal DependencyProperty.OverrideMetadata() method on a per-type basis
+	/// -- that's why the type parameter is required.
+	///
+	/// A classic example of that is the Button.VerticalAlignmentProperty which is overridden
+	/// to be VerticalAlignment.Center, while the default value is VerticalAlignment.Stretch
+	/// from the base class.
+	/// </remarks>
+	public static object? GetDefaultValue<T>(DependencyProperty dependencyProperty)
+		where T : DependencyObject
+		=> dependencyProperty.GetMetadata(typeof(T))?.DefaultValue;
+
+	/// <summary>
+	/// Get the default value of a <see cref="DependencyProperty"/> for the owner type. 
+	/// </summary>
+	/// <remarks>
+	/// This is the value defined in the metadata of the property, which _may_ be overriden
+	/// using the internal DependencyProperty.OverrideMetadata() method on a per-type basis
+	/// -- that's why the type parameter is required.
+	///
+	/// A classic example of that is the Button.VerticalAlignmentProperty which is overridden
+	/// to be VerticalAlignment.Center, while the default value is VerticalAlignment.Stretch
+	/// from the base class.
+	///
+	/// This overload will return the default value for the owner type of the property - where
+	/// the value is originally defined.
+	/// </remarks>
+	public static object? GetDefaultValue(DependencyProperty dependencyProperty)
+		=> dependencyProperty.GetMetadata(dependencyProperty.OwnerType)?.DefaultValue;
+
+	/// <summary>
+	/// Get a reference to <see cref="DependencyProperty"/> by its name and owner type.
+	/// </summary>
+	/// <remarks>
+	/// The name will match the name of the property as defined when the property is registered.
+	///
+	/// There is usually NO "Property" suffix on that name since it's the name that is used in XAML.
+	///
+	/// The name is case-sensitive.
+	/// </remarks>
+	public static DependencyProperty? GetDependencyPropertyByName(Type ownerType, string propertyName)
+		=> DependencyProperty.GetProperty(ownerType, propertyName);
```

Comment:
Same here. It's already easy to access via reflection. Actually, it will be **easier** to access it when it's on `DependencyProperty` when using unsafe accessor. Because currently unsafe accessor can't access something in a static class